testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -294,6 +295,14 @@ public void testDeleteAccessControl()
         assertAccessAllowed("DELETE FROM orders", privilege("orders", SELECT_COLUMN));
     }
 
+    @Test
+    public void testUpdateAccessControl()
+    {
+        assertAccessDenied("UPDATE orders SET orderkey=123", "Cannot update columns \\[\\[orderkey]] in table *.*.*", privilege("orders", UPDATE_TABLE));
```

Comment:
Don't you need to escape closing brackets `\\]` too?

May be `*.*.*` -> `.*` is enough?

Reply:
No, escaping `]` is not required.
As of `*.*.*` you are right

Reply:
> No, escaping ] is not required.

Do you know why? Any references?

Reply:
I was never a regex master. I am sure some nice people on stack overflow know the answer

Reply:
Why the `[[` in the message?

Reply:
That is the whole point of this PR. The `[` are doubled. This commit is before the fix, so I check for double

Reply:
> Do you know why? Any references?

Because if you don't have preceding `[` then `]` in regex means only `]`.